Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/Vector2.h

```diff
@@ -70,3 +71,5 @@ namespace ScriptCanvas
         AZStd::tuple<Vector2Type, NumberType> DirectionTo(const Vector2Type from, const Vector2Type to, NumberType optionalScale = 1.f);
     } // namespace Vector2Functions
 } // namespace ScriptCanvas
+
+#include <Include/ScriptCanvas/Libraries/Math/Vector2.generated.h>
```

Comment:
missing newline at the end of this file
```suggestion
#include <Include/ScriptCanvas/Libraries/Math/Vector2.generated.h>

```